fifa_draft/utils.py

```diff
@@ -72,7 +76,7 @@ def draw_draft_order(group):
     group.picking_history = "Draft started!:"
 
 
-def group_validation(request, group):
+def group_validation(request: WSGIRequest, group: Group) -> messages:
```

Comment:
Nieprawda :stuck_out_tongue: ta funkcja tego nie zwraca

Funkcja zwraca to co jest po `return`, a jak returna nie ma, to znaczy, że zwraca `None`. Jeżeli przypiszesz to do zmiennej, np. `dupa = group_validation(request, group)` to `type(dupa)` to będzie `None`.

To że logujesz tam jakiś `messages.error` to nic nie znaczy, bo tego nie zwracasz jako output funkcji.